docs/source/credits.rst

```diff
@@ -16,7 +16,7 @@ We would like to thank the following sponsors for their financial support in the
 
 **Citation**
 
-Please use the following Bibtex citation when referencing this software. :cite:`sreehari2016plug` (coming soon) :cite:`venkatakrishnan2013plug`
+Please use the following bibtex citation when referencing this software. :cite:`sreehari2016plug` (coming soon) :cite:`venkatakrishnan2013plug`
 ::
 
     @Misc {mbirjax-2024,
```

Comment:
This reference seems to be a hybrid of a reference for a paper describing the new algorithm (title) and for the software itself (other entries). I assume the intent is to create two distinct entries. If this one is for the software, the title should match the software name.

Reply:
Yes, this was just left over text. I moved it to the correct location and I removed the extra references from the bibliography. We don't yet have anything to cite on the VCD. I'm giving a poster at CVPR. Maybe I can cite that abstract once it's done.